### PR TITLE
Speakers Module Complete

### PR DESCRIPTION
**Pull request by sumit-badsara**
## Speaker Module Complete with following API endpoints:
- `speakers/list/<year>`
- `speakers/generate_spreadsheet/`
- `speakers/add_new/`

### CODE REVIEW

File: speakers/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    company         = models.CharField(max_length=128)
+    experience      = models.IntegerField(default=0)
+    email           = models.EmailField(max_length=64, unique=True)
+    description     = models.TextField(blank=True, null=True)
+    verified        = models.BooleanField(default=False)
+    contact         = models.CharField(max_length=10)
+    profile_pic     = models.ImageField(upload_to='static/uploads/speakers',
+                                    null=True, blank=True)
+    social_media    = models.TextField(null=True, blank=True)
+    year            = models.IntegerField(default=2019)
```
Comment (kdpisda) [inline speakers/models.py:15]: RIP Spacing?
Add timestamps

File: speakers/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    
+    def __str__(self):
+        return self.name+" ("+self.company+")"
```
Comment (kdpisda) [inline speakers/models.py]: Better to format string
str("{} {}".format(self.name, self.company))

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "frontend",
```
Comment (kdpisda) [inline frontend/package.json]: Seriously @naveennvrgup ?

File: frontend/public/index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tag above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (kdpisda) [inline frontend/public/index.html]: React App?

File: frontend/src/Home/about.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const about=()=> (<div className="about">
```
Comment (kdpisda) [inline frontend/src/Home/about.js]: Better make a react component

File: frontend/src/Home/glimpses.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import {g1} from '../Images/g1.jpeg';
+import {g2} from '../Images/g2.jpeg';
+import {g3} from '../Images/g3.jpeg';
+import {g4} from '../Images/g4.jpeg';
+import {g5} from '../Images/g5.jpeg';
+import {g6} from '../Images/g6.jpeg';
+import {g7} from '../Images/g7.jpeg';
+import {g8} from '../Images/g8.jpeg';
+import {g9} from '../Images/g9.jpeg';
+import {g10} from '../Images/g10.jpeg';
```
Comment (kdpisda) [inline frontend/src/Home/glimpses.js]: Is it temporary or we are gonna use that?

File: .gitignore [hunk 7]
```diff
@@ -1,4 +1,15 @@
+#Virtual Environments
 env
+venv
+virtualenv
+ENV
+VENV
+
```
Comment (kdpisda) [inline .gitignore]: @BadduCoder how many env folders?

File: speakers/models.py (hunk 1 above)
Comment (kdpisda) [reply speakers/models.py:15]: @BadduCoder ?

**Review by kdpisda — CHANGES_REQUESTED** [review]
Update the PR before it is merged

**Comment by naveennvrgup** [comment]
closed because it not relevant.

**Pull request closed without merging**